src/emmy/abstract/function.cljc

```diff
@@ -311,3 +301,11 @@
               (f/arity f)
               (domain-types f)
               (range-type f)))
+
+(defmethod g/zero-like [::function] [^Function a] (fn [& _] (g/zero-like (.-range a))))
+(defmethod g/one-like [::function] [^Function a] (fn [& _] (g/one-like (.-range a))))
+(defmethod g/identity-like [::function] [^Function a]
+  (let [meta {:arity (.-arity a) :from :identity-like}]
+    (with-meta identity meta)))
+(defmethod g/exact? [::function] [a] (f/compose g/exact? a))
```

Comment:
hmm, won't this always return symbols, which will register as false? I guess that makes sense!